src/base/bittorrent/session.cpp

```diff
@@ -379,6 +379,7 @@ Session::Session(QObject *parent)
     , m_checkingMemUsage(BITTORRENT_SESSION_KEY("CheckingMemUsageSize"), 32)
     , m_diskCacheSize(BITTORRENT_SESSION_KEY("DiskCacheSize"), -1)
     , m_diskCacheTTL(BITTORRENT_SESSION_KEY("DiskCacheTTL"), 60)
+    , m_queuedDiskBufferSize(BITTORRENT_SESSION_KEY("QueuedDiskBufferSize"), (1024 * 1024))
```

Comment:
`QueuedDiskBufferSize` sounds bad for me. Maybe `DiskQueueSize`?

Reply:
Done.